### DISCUSSION TITLE
Revised/New Edition TPB Handling

### DISCUSSION THREAD

**Discussion by zanish** [bpepple/metron#318] [Q&A] [answered]
I know this doesn't happen often but I am starting to add Hellblazer TPBs and there are the original and new editions. You can see the differences here: https://en.wikipedia.org/wiki/List_of_Hellblazer_comics#Collected_editions, but they are materially different than the OG release.

For organization would it be best to either:
a. Make 1 series and add both versions of the TPB to that series
b. Make 2 series (Hellblazer TPB & Hellblazer New Edition TPB) and add the respective tpbs to each series? 

I didn't see anything on the editing guide and don't want to go rogue.

Thanks!

**Comment by bpepple** [comment]
Hmmm... My initial thought is to make two series since they reprint different material.

The problem I see, is something I've been giving some thought about recently, is the first set of TPB's don't have a volume numbers (hence no Vol 2, etc.) and were published in no logical order, so I'm wondering if we should have those as individual series, like **Hellblazer: Original Sins**, and not how we tend to handle TPB like the second set, which we add as a single series using the volume number like a ongoing series' issue number.

I need to give this a little more thought, and have some other folks weigh in on it before making a final decision.

**Comment by bpepple** [comment, marked as answer]
Alright, gave this some thought and no one in the Matrix channel had a strong opinion, so I think it should be handled as:

1. The original TPB's should be created with a separate series per TPB, since they all are a Vol. 1 and were printed without regard to issue order. For example, `John Constantine, Hellblazer: The Devil You Know #1`
2. The new edition TPB should be created as one series, with the volume acting like an issue number, similar to how [GCD](https://www.comics.org/series/80455/) does.

If you have any questions, or something needs to be better clarified don't hesitate to ask.
